docs/API.md

```diff
@@ -83,6 +85,8 @@ Returns a new [logger](#logger) instance.
   timestamp to ISO 8601 date format, and reserialize the JSON (equivalent to `pino -t`).
   * `formatter` (function): a custom function to format the line, is passed the
   JSON object as an argument and should return a string value.
+  * `levelFirst` (boolean): if set to `true`, it will print the name of the log
+  level as the first field in the log line. Default: `false`.
```

Comment:
I may be missing something obvious, but I don't see this referenced in the code.